### PR TITLE
Fix incorrect types

### PR DESCRIPTION
**Pull request by Hannes-Sandahl-Mpya**
Fix incorrect type of the 'type' prop of ContentCard
Fix incorrect return type for getContentCards

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -419,9 +414,9 @@ export function logContentCardsDisplayed(): void;
 
 /**
  * Returns a content cards array
- * @returns {Promise<ContentCard[]>}
+ * @returns {Promise<(ClassicContentCard | BannerContentCard | CaptionedContentCard)[]>}
  */
-export function getContentCards(): Promise<ContentCard[]>;
+export function getContentCards(): Promise<(ClassicContentCard | BannerContentCard | CaptionedContentCard)[]>;
```
Comment (wesleyorbin) [inline index.d.ts:419]: I actually think we could fix the typing issues here by instead converting the three `ContentCard` subclasses from `interface` to `class`. That should allow `instanceof` to work correctly. Would you be able to quickly verify if that works for you?

Comment (Hannes-Sandahl-Mpya) [reply index.d.ts:419]: Hi there and thanks for the quick feedback!

Hmm not really sure I follow. For me classes are object factories while type aliases are contracts describing a _shape_ that values have, and that's what we're doing here in the typings file. So don't really see the use case for classes here

Comment (wesleyorbin) [reply index.d.ts:419]: Sorry, that's an error on my part. While those subclasses _are_ objects in the underlying native SDKs, you are correct that we only care about the contract here.

Thank you for the contribution! We'll update it in our private source and include it in the next release.

**Comment by davidbielik** [comment]
Hi @Hannes-Sandahl-Mpya thanks again for raising this. the type definitions you suggested were merged to our private source last year shortly after @wesleyorbin's last comment.

**Pull request closed without merging**